### PR TITLE
Angular ui-router - blog post

### PR DESCRIPTION
**Pull request by YaronMiro**
#127

### CODE REVIEW

**Comment by YaronMiro** [comment]
@amitaibu 
Ready for review :)

File: app/_posts/2015-4-26-angularjs-ui-router.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  <img src="{{BASE_PATH}}/assets/images/posts/angularjs-ui-router/demo.gif">
+  <div class="caption">Itunes top 30 movie angularJS + Ui-router demo application</div>
+</div>
+
+<!-- more -->
+
+## What is the idea behind Ui-router ?
+Ui-router has the ability to structure your application upon states.
+These states can be [parallel](https://github.com/angular-ui/ui-router/wiki#state-manager), [nested](https://github.com/angular-ui/ui-router/wiki/Nested-States-%26-Nested-Views) and even structured into [multiple named views](https://github.com/angular-ui/ui-router/wiki/Nested-States-%26-Nested-Views).
+a state is <u> not dependent </u> upon a url, You can assign a url to a state but it can also be transitioned from one to another <u>without</u> a url change.
```
Comment (amitaibu) [inline app/_posts/2015-4-26-angularjs-ui-router.md]: _underline_

Comment (YaronMiro) [inline app/_posts/2015-4-26-angularjs-ui-router.md]: @amitaibu 
Text changes to an ```<em>```which is an italic style element not underline text.
We could change the the ```CSS``` of that element, But I think it's not the correct way.
May I leave it with the ```<U>``` element ?

Comment (amitaibu) [inline app/_posts/2015-4-26-angularjs-ui-router.md]: no need for underline, em is enough.

**Comment by amitaibu** [comment]
@niryariv ready for review.

**Comment by amitaibu** [comment]
I removed the gif as it was 12.5MB

Merged and deployed - http://www.gizra.com/content/angularjs-ui-router/
Thanks @YaronMiro and Mazal Tov ;)

**Pull request closed without merging**